conf/deploy.json

```diff
@@ -6,6 +6,16 @@
             "data": {
                 "bucket": "content-api-dist"
             }
+        },
+        "content-api-dashboard-ami-update" : {
+            "type": "ami-cloudformation-parameter",
+            "data": {
+                "amiTags" : ["ubuntu-wily-java-8"],
```

Comment:
The `amiTags` parameter is a map from tag name to value. (I only worked this out by reading [the source](https://github.com/guardian/riff-raff/blob/master/magenta-lib/src/main/scala/magenta/deployment_type/AmiCloudFormationParameter.scala#L23))

So I think it should be `"amiTags": { "RecipeId": "ubuntu-wily-java-8" }